packages/hooks/src/hooks/useTokenBalance/index.ts

```diff
@@ -0,0 +1 @@
+export { useTokenBalance } from './useTokenBalance';
```

Comment:
i think you also need to export this from the root hooks directory here: https://github.com/Developer-DAO/web3-ui/blob/master/packages/hooks/src/index.ts